### PR TITLE
Fix mpm package name for Ubuntu Xenial

### PR DESCRIPTION
**Pull request by HelenCampbell**

### CODE REVIEW

File: manifests/mpm.pp [hunk 1]
```diff
@@ -66,12 +66,18 @@
...
 
+      if $mpm == 'itk' and $::operatingsystem == 'Ubuntu' and $::operatingsystemrelease == '16.04' {
+        $packagename = 'libapache2-mpm-itk'
+      } else {
+        $packagename =  "apache2-mpm-${mpm}"
+      }
+
       if versioncmp($apache_version, '2.4') < 0 or $mpm == 'itk' {
-        package { "apache2-mpm-${mpm}":
+        package { "$packagename":
```
Comment (hunner) [inline manifests/mpm.pp]: These two variables will produce lint errors of quoted variables so should be unquoted.

Comment (bmjen) [inline manifests/mpm.pp]: +1

**Comment by bmjen** [comment]
Other than @hunner suggestion, LGTM.

**Comment by HelenCampbell** [comment]
@bmjen @hunner Better?

**Comment by bmjen** [comment]
👍

**Comment by tphoney** [comment]
@HelenCampbell this is a lint issue 👍

**Comment by bmjen** [comment]
@HelenCampbell I believe the pcci failures are also related to the extra space, in that the variable now has artifact characters prepended to it. Maybe strict variables related?

**Comment by tphoney** [comment]
#1477 supersedes this PR.

**Pull request closed without merging**